src/components/20-molecules/list/index.js

```diff
@@ -0,0 +1,80 @@
+import { html, css, unsafeCSS } from 'lit-element';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html';
+import {
+  defineVersioned,
+  /* versionedHtml, */
```

Comment:
Why commented out?

Reply:
create-aom default. I was not sure if I need that for versioning of the childelements, but I can actually remove it, since the child elements are user issued.